bpf/cpu/shared.h

```diff
@@ -53,7 +53,7 @@ struct {
 
 struct {
     __uint(type, BPF_MAP_TYPE_HASH);
-    __uint(max_entries, 64000);
+    __uint(max_entries, 1); // default: 64000. Set in the user-space.
```

Comment:
nit: As the max entries value when enabled lives in userspace, should we remove the "default: 64000"? It's one more thing to keep in sync, a small one, though 😄